### PR TITLE
[1.0.0] Changelog updated

### PR DESCRIPTION
**Pull request by loick**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blablacar/ui-library",
-  "version": "0.24.1",
+  "version": "1.0.0",
```
Comment (Tuxosaurus) [inline package.json]: Should we update the Readme by making clear this is still rather an alpha version?

Comment (loick) [reply package.json]: Why is it an alpha version ?

Comment (Tuxosaurus) [reply package.json]: I think we're missing to many "guidelines" and explanations (and support) to leave this out in the wild.

Comment (Tuxosaurus) [reply package.json]: Ok as discussed, adding "guidelines" is planned.

**Pull request merged**